Directory.Build.props

```diff
@@ -37,7 +37,7 @@
     <!-- Issue: https://github.com/dotnet/aspire/issues/8488 -->
     <!-- xUnit1051: Calls to methods which accept CancellationToken should use TestContext.Current.CancellationToken to allow test cancellation to be more responsive. -->
     <!-- TODO: Re-enable and remove this. -->
-    <NoWarn>$(NoWarn);xUnit1051</NoWarn>
+    <NoWarn>$(NoWarn);xUnit1051;CS0162;CS1591;CS9113;IDE0059;IDE0051;IDE2000;IDE0005</NoWarn>
```

Comment:
Building new API and experimenting is horribly difficult because of these (and many more) "warnings as errors".
I'll revert this when I get to the finish line.

Reply:
Opps, I forgot to rollback this. I'll fix it in a follow up.

Reply:
Did you revert this?

Reply:
https://github.com/dotnet/aspire/pull/9138